src/Symfony/Bundle/SecurityBundle/Security.php

```diff
@@ -151,4 +163,11 @@ private function getAuthenticator(?string $authenticatorName, string $firewallNa
 
         return $firewallAuthenticatorLocator->get($authenticatorId);
     }
+
+    public static function getSubscribedServices(): array
+    {
+        return [
+            'security.user_authorization_checker' => UserAuthorizationCheckerInterface::class,
```

Comment:
this should list them all :)
see `service_locator()` in `services.php`

Reply:
I think I've done this correctly.